src/YAxisLabels.js

```diff
@@ -143,17 +143,12 @@ class YAxisLabels extends React.Component {
      * `mouseclick` event handler callback, called when user's mouse clicks the label.
      */
     onMouseClickLabel: PropTypes.func,
-    /**
-     * Format to use for the labels or accessor that returns the updated label.
-     *
-     * For example, given labels with real numbers one can pass in 0.[0] to round to the first significant digit.
-     */
-    format: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
     /**
      * Formats to use for the labels in priority order. XAxisLabels will try to be smart about which format
-     * to use that keeps the labels distinct and provides the least amount of collisions when rendered.
+     * to use that keeps the labels distinct and provides the least amount of collisions when rendered. Formats
+     * can be either string(s) or function(s), utilizing d3-format.
      *
-     * For example, given labels with real numbers one can pass in 0.[0] to round to the first significant digit
+     * For example, given labels with real numbers one can pass in [".0%"] for a rounded percentage, like 12%.
```

Comment:
Thanks for updating the documentation! Let's also add that we've made the decision to use `d3.format` as our format function :)

Reply:
This is mentioned in the line above.